jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java

```diff
@@ -354,15 +354,14 @@ public void setIdentityService(IdentityService identityService) {
     }
 
     /**
-     * Validate a user identity. Validate that a UserIdentity previously created by a call to
-     * {@link #login(String, Object)} is still valid.
+     * Validate a user identity. Validate that a UserIdentity is still valid.
```

Comment:
all good points.  I was just doing minimal edits to remove the hard javadoc errors of broken links.